src/services/healthcareProfessionalService.ts

```diff
@@ -1,13 +1,53 @@
-import { healthcareProfessionals } from '../mockData/mockData';
-import { HealthcareProfessional } from '../typeDefs/gqlTypes';
+import { HealthcareProfessional, LocaleName } from '../typeDefs/gqlTypes';
+import { HealthcareProfessional as PrismaHealthcareProfessional,
+    LocaleName as PrismaLocaleName } from '@prisma/client';
+
+import prisma from '../db/client';
+
+type HealthcareProfessionalAndRelations = (PrismaHealthcareProfessional & 
+    { names: PrismaLocaleName[]})
+
+function convertPrismaToGqlHealthcareProfessional(input: HealthcareProfessionalAndRelations) {
```

Comment:
idea for improving readability: not sure if we need this level of abstraction yet, but typically I put mappings into a separate folder to keep the services a bit cleaner.

Reply:
It's a good idea, can we save for a future refactor?

Reply:
yup! whenever you feel like it's starting to make the files long and hard to follow is probably the right time 😸